api/src/main/resources/workbench-api.yaml

```diff
@@ -5876,6 +5876,8 @@ definitions:
   AdminWorkspaceCloudStorageCounts:
     type: object
     properties:
+      storageBucketPath:
+        type: string
```

Comment:
Maybe include a short description here describing how this the absolute URI (including the bucket name) and the relative path. That's pretty clear though.

Reply:
Yep – done.